### DISCUSSION TITLE
Teams integration with over 500 channels

### DISCUSSION THREAD

**Discussion by arno-jost** [Erudika/scoold#378] [Q&A]
As a member of a large tenant, the configuration of the team integration in our scoold instance leads to about 600 rooms that are automatically created and drive the performance of scoold to unusable levels.

Is there a trick to limit the list of channels for the sync already in the config?

**Comment by albogdano** [comment]
Scoold has two options for mapping channels to Scoold spaces:
```
scoold.teams.map_workspaces_to_spaces = true
scoold.teams.map_channels_to_spaces = false
```
How did you configure these? I guess the second option in your case is set to `true`.
Once the Scoold spaces have been created, you can only manually delete them as they won't automatically disappear after you change the configuration.
